presto-cli-provisio/src/main/provisio/presto.xml

```diff
@@ -0,0 +1,10 @@
+<runtime>
+    <!-- Target -->
+    <archive name="${project.artifactId}-${project.version}.tar.gz" />
+
+    <!-- Presto CLI executable JAR -->
+    <!-- TODO: This should not add dependency jars into the archive -->
```

Comment:
@jvanzyl Is there any way to tell provisio to only include the exact artifact specified without pulling in any dependencies?